src/services/process-service.ts

```diff
@@ -52,7 +52,7 @@ export default class ProcessService implements Disposable {
     const cmlPattern = /^([0-9]+)\s+([0-9]+)\s(.+$)/;
 
     // build and execute command line tool "ps" to get details of all running processes
-    const args = ["-o pid,ppid,command"];
+    const args = ["-e -o pid,ppid,command"];
```

Comment:
do we really need -e switch? it is going to print all processes outside of what we have forked/spawned from the extensions
<img width="635" alt="Screenshot 2022-05-30 at 9 10 52 AM" src="https://user-images.githubusercontent.com/653805/170900216-83a8d5f7-5efb-4d8f-b59a-356977a06a80.png">

process-service prints all processes under current user:
<img width="309" alt="image" src="https://user-images.githubusercontent.com/653805/170900406-3723fd8d-adb1-4879-b670-d1da2dc70d1b.png">

running `ps -eo ppid,pid,command,user | more`

<img width="390" alt="image" src="https://user-images.githubusercontent.com/653805/170900482-bce0d91d-bbfb-48ab-98d1-91b6ca707c1b.png">

We get all processes, which will be slower for the extension, since we are scanning all process (which happens every 1 second)